### ISSUE TITLE
Dev test: Ohms Law 1.4.0-dev.26

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/QA#289]
@emily-phet, @terracoda, @jbphet, @zepumph, Ohms Law 1.4.0-dev.26 is ready for dev testing. Document issues in https://github.com/phetsims/ohms-law/issues and link to this issue.

This sim includes accessibility features, so please test those as part of this dev test. This sim 
includes support for a new screen reader, VoiceOver with mobile Safari. VoiceOver with mobile safari works differently than other screen readers. I have included information about how to use it in https://phet-dev.colorado.edu/html/jg-tests/reader-intro.html, but let me know if you have any other questions. Before testing the simulation with VoiceOver, please read through that document and make sure that you can interact with all of the sliders and buttons on that page.

Assigning @ariel-phet for prioritization.

Thank you!

<details>
<summary><b>General Dev Test</b></summary>

<h3>What to Test</h3>

- Click every single button.
- If there is sound, make sure it works.
- Make sure you can't lose anything.
- Play with the sim normally.
- Try to break the sim.

<h3>Focus and Special Instructions</h3>

[Provide further instructions here. If you have any further tests you want done or specific platforms you want tested, list them here. Any behaviors you want QA to pay special attention to should be listed here.]

<h3>General Dev Test Platforms</h3>

- [x] Latest macOS, Chrome and Safari
- [x] Latest iOS, Safari
- [x] Windows 10, all browsers
- [x] Latest Chrome OS, Chrome

<h3>Link(s)</h3>

- **[Simulation](https://phet-dev.colorado.edu/html/ohms-law/1.4.0-dev.26/phet/ohms-law_en_phet.html)**

<hr>
</details>

<details>
<summary><b>Accessibility (a11y) Dev Test</b></summary>

<h3>What to Test</h3>

- Specific instructions can be found below.
- Make sure the a11y feature doesn't negatively affect the sim in any way.

<h3>Focus and Special Instructions</h3>

[Provide further instructions here. If you have any further tests you want done or specific platforms you want tested, list them here. Any behaviors you want QA to pay special attention to should be listed here.]

<h3>Link(s)</h3>

- **[a11y View](https://phet-dev.colorado.edu/html/ohms-law/1.4.0-dev.26/phet/ohms-law_a11y_view.html)**
- **[Simulation](https://phet-dev.colorado.edu/html/ohms-law/1.4.0-dev.26/phet/ohms-law_en_phet.html)**

<h3>Screen Readers</h3>

This sim supports screen readers. If you are unfamiliar with screen readers, please ask Katie to introduce you to screen readers. If you simply need a refresher on screen readers, please consult the [QA Book](link), which should have all of the information you need as well as a link to a screen reader tutorial made by Jesse. Otherwise, look over the a11y view before opening the simulation. Once you've done that, open the simulation and make sure alerts and descriptions work as intended.

<h3>Platforms and Screen Readers to Be Tested</h3>

- [x] Windows 10 + Latest Chrome + Latest JAWS
- [x] Windows 10 + Latest Firefox + Latest NVDA
- [x] macOS + Safari + VoiceOver
- [x] iOS + Safari + VoiceOver (only if specified in testing issue)

<h3>Critical Screen Reader Information</h3>

We are tracking known screen reader bugs in [this Google Document](https://drive.google.com/open?id=12kTs-g-iKEIH1dyG7Q41_W_oNL4gUKbkW-IQgZjMUBw). If you find a screen reader bug, please check it against this list.

<h3>Keyboard Navigation</h3>

This sim supports keyboard navigation. Please make sure it works as intended on all platforms by itself and with a screen reader.

<hr>

</details>

<details>
<summary><b>FAQs for QA Members</b></summary>

<br>
  
  

  <details>
  <summary><i>There are multiple tests in this issue... Which test should I do first?</i></summary> 
  
  Test in order! Test the first thing first, the second thing second, and so on.

  </details>

  <br>

  

  <details>
  <summary><i>How should I format my issue?</i></summary>
  
  Here's a template for making issues:

      <b>Test Device</b>

      blah

      <b>Operating System</b>

      blah

      <b>Browser</b>

      blah

      <b>Problem Description</b>

      blah

      <b>Steps to Reproduce</b>

      blah

      <b>Visuals</b>

      blah

      <details>
      <summary><b>Troubleshooting Information</b></summary>

      blah

      </details>

  </details>

  <br>

  

  <details>
  <summary><i>Who should I assign?</i></summary>
  
  We typically assign the developer who opened the issue in the QA repository.

  </details>
  
  <br>

  

  <details>
  <summary><i>My question isn't in here... What should I do?</i></summary>
  
  You should:

  1. Consult the [QA Book](link).
  2. Google it.
  3. Ask Katie.
  4. Ask a developer.
  5. Google it again.
  6. Cry.

  </details>

<br>

<hr>

</details>

**Comment by jessegreenberg** [comment]
And just a heads up, we are already aware of some mobile VoiceOver quirks! Beware of `#21` on this list as you test. https://docs.google.com/document/d/12kTs-g-iKEIH1dyG7Q41_W_oNL4gUKbkW-IQgZjMUBw/edit

**Comment by ariel-phet** [comment]
Goal is to publish this sim by end of March, so good to have the dev test take high priority

**Comment by KatieWoe** [comment]
QA is done

**Comment by jessegreenberg** [comment]
Thanks very much @KatieWoe, thats great.
